src/Microsoft.AspNet.Html.Abstractions/IHtmlContentBuilder.cs

```diff
@@ -29,7 +29,7 @@ public interface IHtmlContentBuilder : IHtmlContent
         /// </summary>
         /// <param name="encoded">The HTML encoded <see cref="string"/> to append.</param>
         /// <returns>The <see cref="IHtmlContentBuilder"/>.</returns>
-        IHtmlContentBuilder AppendEncoded(string encoded);
+        IHtmlContentBuilder AppendHtml(string encoded);
```

Comment:
Should the documenation and parameter name change? example: `AppendHtml("<b>")` ...the string is not encoded but raw html...
Same comment in all other relevant places.